homeassistant/components/plugwise/gateway.py

```diff
@@ -42,6 +50,7 @@ async def async_setup_entry_gw(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     api = Smile(
         host=entry.data[CONF_HOST],
+        username=entry.data.get(CONF_USERNAME, DEFAULT_USERNAME),
         password=entry.data[CONF_PASSWORD],
         port=entry.data.get(CONF_PORT, DEFAULT_PORT),
         timeout=30,
```

Comment:
should the default timeout be used here as well? can be done in new PR if so.

Reply:
We'll have a look at this, thanks!